src/actions/api.js

```diff
@@ -11,100 +11,56 @@ export const getArticle = (article_id) => {
 };
 
 export const getArticles = () => {
-	const promise = new Promise((resolve, reject) => {
-		axios
-			.get(`${ENDPOINT}/articles`)
-			.then((res) => {
-				resolve(res.data);
-			})
-			.catch((err) => {
-				reject(err);
-			});
+	return axios.get(`${ENDPOINT}/articles`).then((res) => {
+		return res.data;
 	});
+};
 
-	return promise;
+export const getSortedArticles = (sortBy) => {
```

Comment:
This can be combined with getArticles as they are both essentially doing the same thing. Have a look at the notes on how to use axios params

Reply:
https://notes.northcoders.com/courses/js-front-end/react-projects